uat/src/main/resources/greengrass/features/component.feature

```diff
@@ -1,19 +0,0 @@
-Feature: Testing Cloud component in Greengrass
-
-    Background:
```

Comment:
Removed all this dummy tests that were just used for the initial setup of the project.